runtime/vara/src/integration_tests.rs

```diff
@@ -543,91 +544,6 @@ fn dust_ends_up_in_offset_pool() {
         });
 }
 
-#[test]
```

Comment:
why?

Reply:
there is no ability to propose anything directly after new treasury changes. Test is unreliable.